shared/test/common_test_helper.rb

```diff
@@ -80,7 +80,13 @@ def around(&block)
     VCR.use_cassette(cassette_name, record: record_mode) do
       PEGASUS_DB.transaction(rollback: :always) do
         DASHBOARD_DB.transaction(rollback: :always) do
+          # Use Minitest#stub here even though we generally prefer Mocha#stubs.
+          # Mocha keeps its stubbing logic simple in an attempt to avoid
+          # overcomplicating tests, but in this case we specifically do need a
+          # dynamic return value, which Mocha does not support.
+          # rubocop:disable CustomCops/PreferMochaStubsToMinitestStub
           AWS::S3.stub(:random, proc {random.bytes(16).unpack1('H*')}, &block)
+          # rubocop:enable CustomCops/PreferMochaStubsToMinitestStub
```

Comment:
[Yep!](https://github.com/code-dot-org/code-dot-org/pull/55172/files#diff-bf5a49ae56b4c04a6e23c087f6bc12372e033769c574137cde3df3465d202606)

I'm planning to add it as a follow-up to this PR. I certainly could combine the two PRs if you think that would be cleaner; I just wanted to give folks the opportunity to consider separately the questions of just updating these existing examples versus putting in place long-term enforcement of a particular pattern.

Reply:
It could also make sense to keep these new directives out of this PR and only add them in the PR that also adds the rule. I included it here so that this PR captures all the ways we are currently using Minitest's `stub` in one place, for easy reference.

Reply:
either way is fine with me!